### PR TITLE
Docs: Link the manpage of undelying functions

### PR DESCRIPTION
**Pull request by rusty-snake**

### CODE REVIEW

**Comment by rusty-snake** [comment]
No idea why the CI is failing.

File: libseccomp/src/api.rs [hunk 1]
```diff
@@ -16,6 +16,9 @@ use libseccomp_sys::*;
 ///
 /// [seccomp_api_get(3)]: https://www.man7.org/linux/man-pages/man3/seccomp_api_get.3.html
 ///
+/// This function is corosponds to
```
[the remark this answers is not in the document]
Comment (rusty-snake) [reply libseccomp/src/api.rs:19]: Thanks for the review.

What do you mean exactly?

File: libseccomp/src/filter_context.rs [hunk 2]
```diff
@@ -586,6 +652,9 @@ impl ScmpFilterContext {
     ///
     /// Defaults to on (`state` == `true`).
     ///
+    /// This function is corosponds to
+    /// [`seccomp_`]().
```
[the remark this answers is not in the document]
Comment (rusty-snake) [reply libseccomp/src/filter_context.rs:656]: Overlooked, will fix to

```suggestion
    /// [`seccomp_attr_set`](https://man7.org/linux/man-pages/man3/seccomp_attr_set.3.html).
```

File: libseccomp/src/api.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (ManaSugi) [reply libseccomp/src/api.rs:19]: `This function is corosponds to` -> `This function corresponds to`
Sorry I don't know `be corosponds to`.

[the remark this answers is not in the document]
Comment (rusty-snake) [reply libseccomp/src/api.rs:19]: Uhh yes thats right

**Comment by ManaSugi** [comment]
LGTM! Please squash the commits.

**Comment by ManaSugi** [comment]
@rusty-snake LGTM. Thank you!

**Review by ManaSugi — APPROVED** [review]

**Pull request merged**